OtripleS.Web.Api/Models/CourseAttachments/Exceptions/LockedCourseAttachmentException.cs

```diff
@@ -0,0 +1,15 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System;
+
+namespace OtripleS.Web.Api.Models.CourseAttachments.Exceptions
+{
+    public class LockedCourseAttachmentException : Exception
+    {
+        public LockedCourseAttachmentException(Exception innerException)
+            : base("Locked CourseAttachment record exception, please try again later.", innerException) { }
```

Comment:
These messages are meant for end users to read.
This should be Course Attachment not CourseAttachment